third_party/terraform/resources/resource_cloudfunctions_function.go

```diff
@@ -166,6 +182,22 @@ func resourceCloudFunctionsFunction() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"ingress_settings": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
`ForceNew` typically means destroy/create if changed instead of updating in-place, but I see you added support for this field in the Update() func. Which is it?

Reply:
oh maybe i didn't understand it quite well. Background for this is, if we change ingress_settings gcp will restart the cloudfunction. I thought if the field yields to an restart this parameter needs to be set. but i can leave it out.